P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -347,19 +348,40 @@ used to define a polygon of any shape using this coordinate system:-->
             </facsimile></egXML>
       </p>
 
-      <p>In this example a single <gi>graphic</gi> element has been associated directly with the
+
+      <p>In the examples above a single <gi>graphic</gi> element has been associated directly with the
          surface of the page rather than nesting it within a zone. However, it is also possible to
          include multiple <gi>zone</gi> elements which contain a <gi>graphic</gi> element, if for
          example a detailed image is available. Since all <gi>zone</gi> elements use the same
          coordinate system (that defined by their parent <gi>surface</gi>), there is no need to
          demonstrate enclosure of one zone within another by means of nesting. To continue the
-         current example, supposing that we have an additional image called <ident type="file">Bovelles49r-detail.png</ident> containing an additional image of the figure in the
+         current example, supposing that we have an additional image called 
+         <ident type="file">Bovelles49r-detail.png</ident> containing an additional image of the figure in the
          third zone above, we might encode that zone as follows: <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:lang="und" source="#UND">
             <zone ulx="105" uly="76" lrx="175" lry="160">
                <graphic url="Bovelles49r-detail.png"/>
             </zone>
          </egXML>
       </p>
+      
+      
+      <p>Within a <gi>surface</gi> or <gi>zone</gi>, individual lines can be identified using the 
+         <gi>path</gi> element, which also carries the <att>points</att>
+         attribute:
+         <figure xml:id="facs-fig3">
+            <graphic url="Images/facs-fig3.jpg"/>
+            <head>Smaller detail of p 49r from Bovelles highlighting two specific lines</head>
+         </figure>
+         <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <surface ulx="0" uly="0" lrx="443" lry="272">
+               <graphic url="facs-fig3.jpg"/>
+               <path points="74,73 171,244" xml:id="balan"/>
+               <path points="71,203 173,116" xml:id="dindan"/>
+            </surface></egXML>
+         
+         This is useful in cases where you need to link an annotation or explanation to a specific line on
```

Comment:
I would recommend avoiding using 'you' in the Guidelines. Change this to something like "This is useful for linking an annotation or explanation to..."

Reply:
I don't mind making that change, but to be honest I see nothing wrong with using "you" like this. What's the basis of the objection? Is there something in our styleguide about this?